src/sass/layouts/_hero-header.scss

```diff
@@ -1,3 +1,184 @@
 .hero-header {
-  color: rebeccapurple;
+  height: 502px;
+  background-color: var(--hero-primary-background-color);
+  position: relative;
+  overflow: hidden;
+
+  @include for-tablet-up {
+    height: 399px;
+  }
+}
+
+.top-panel {
+  display: flex;
+  align-items: center;
+  padding-top: 6px;
+  margin-bottom: 137px;
+
+  @include for-tablet-up {
+    padding-top: 37px;
+    margin-bottom: 15px;
+  }
+}
+
+.top-panel__icon-menu {
+  stroke: #d41443;
+}
+
+.top-panel__btn {
+  margin-left: auto;
+  padding: 0;
+  background-color: rgba(255, 184, 202, 0);
+
+  @include for-tablet-up {
+    margin-right: 30px;
+    position: relative;
+    z-index: 2;
+  }
+}
+
+.hero {
+  position: relative;
+  z-index: 3;
+
+  @include for-tablet-up {
+    padding-bottom: 28px;
+  }
+}
+
+.hero__title {
+  margin-bottom: 20px;
+
+  font-size: 26px;
+  line-height: 1.15;
+  color: var(--secondary-text-color);
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  text-align: left;
+}
+
+.hero__title--accent {
+  font-size: 48px;
+  line-height: 0.96;
+  color: var(--primary-section-title-color);
+}
+
+.hero__btn {
+  display: flex;
+  align-items: center;
+
+  @include for-tablet-up {
+    margin-bottom: 20px;
+  }
+}
+
+.hero__item {
+  margin-right: 10px;
+}
+
+.hero__item--indent {
+  margin-right: 0;
+}
+
+.hero__link {
+  padding: 7px 17px;
+
+  border-radius: 20px;
+  font-family: DM Sans;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 8px;
+  line-height: 1.25;
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  background-color: var(--secondary-btn-background-color);
+  color: var(--secondary-text-color);
+
+  transition: color var(--animation-time) var(--animation-function),
```

Comment:
you could write transition: get-transition-prop(color), get-transition-prop(background-color);